specification/cat/indices/CatIndicesRequest.ts

```diff
@@ -22,12 +22,21 @@ import { Bytes, ExpandWildcards, HealthStatus, Indices } from '@_types/common'
 import { TimeUnit } from '@_types/Time'
 
 /**
+ * Get index information.
  * Returns high-level information about indices in a cluster, including backing indices for data streams.
- * IMPORTANT: cat APIs are only intended for human consumption using the command line or Kibana console.
- * They are not intended for use by applications. For application consumption, use the get index API.
- * Use the cat indices API to get the following information for each index in a cluster: shard count; document count; deleted document count; primary store size; total store size of all shards, including shard replicas.
+ * > info
+ * > CAT APIs are only intended for human consumption using the command line or Kibana console.
+ * They are not intended for use by applications. For application consumption, use an index endpoint.
+ * 
```

Comment:
these empty lines are failing CI, but newline characters (\n) are still being escaped. investigating

Reply:
played around a little with this. `<br>` is just adding newlines into the note block, not "ending" the note block: 

```
        "description": "Returns high-level information about indices in a cluster, including backing indices for data streams.\n> info\n> CAT APIs are only intended for human consumption using the command line or Kibana console.\nThey are not intended for use by applications. For application consumption, use an index endpoint.<br><br>Use this request to get the following information for each index in a cluster:\n- shard count\n- document count\n- deleted document count\n- primary store size\n- total store size of all shards, including shard replicas\n\nThese metrics are retrieved directly from Lucene, which Elasticsearch uses internally to power indexing and search. As a result, all document counts include hidden nested documents.\nTo get an accurate count of Elasticsearch documents, use the [/_cat/count](#operation-cat-count) or [count](#endpoint-count) endpoints.",
```

<img width="448" alt="image" src="https://github.com/user-attachments/assets/27b1f04f-73a0-4ff9-8b7f-19159142544d">

explicit newline characters are being escaped:

```
        "description": "Returns high-level information about indices in a cluster, including backing indices for data streams.\n> info\n> CAT APIs are only intended for human consumption using the command line or Kibana console.\nThey are not intended for use by applications. For application consumption, use an index endpoint.\\n\nUse this request to get the following information for each index in a cluster:\n- shard count\n- document count\n- deleted document count\n- primary store size\n- total store size of all shards, including shard replicas\\n\nThese metrics are retrieved directly from Lucene, which Elasticsearch uses internally to power indexing and search. As a result, all document counts include hidden nested documents.\nTo get an accurate count of Elasticsearch documents, use the [/_cat/count](#operation-cat-count) or [count](#endpoint-count) endpoints.",
```

<img width="474" alt="image" src="https://github.com/user-attachments/assets/4df8665f-26d2-4c92-b941-04e0e19ce36f">

and explicit spaces are being parsed as `\n\n` into the reference, but failing CI.

<img width="469" alt="image" src="https://github.com/user-attachments/assets/7b7e0abf-8803-49b1-b2e3-f7b90cb61ed1">